src/openzeppelin/token/erc20/library.cairo

```diff
@@ -321,3 +310,50 @@ func _transfer{
     Transfer.emit(sender, recipient, amount)
     return ()
 end
+
+func _approve{
```

Comment:
The name should be prefixed with`ERC20_` to follow the [extensibility pattern](https://github.com/OpenZeppelin/cairo-contracts/blob/main/docs/Extensibility.md#libraries), same goes for `_spendAllowance`

Reply:
i think there's already an ERC20_approve in there, this is the underlying method (like the internal function in solidity). Are there best practices for functions that are supposed to be internal? i know there's currently no decorator but maybe the preceding underscore is enough to convey it

Reply:
@koloz193  +1

We need clear naming convention for public vs internal vs storage

Reply:
> We need clear naming conventions for public vs internal vs storage

Agreed. I think @Amxx's idea of `ERC20_storage_name` might be worth considering.

Reply:
Continued discussion in https://github.com/OpenZeppelin/cairo-contracts/discussions/267